tests/integration/actions.go

```diff
@@ -1236,6 +1236,8 @@ func (tr TestRun) waitForSlashThrottleDequeue(
 
 		time.Sleep(500 * time.Millisecond)
 	}
+	// Sleep 20 seconds to pass a block, allowing the jailing to be incorporated into voting power
```

Comment:
@MSalopek depending on block time when running in the cloud, this may need to be adjusted